x-pack/plugins/monitoring/public/components/elasticsearch/shard_allocation/lib/decorate_shards.js

```diff
@@ -32,7 +32,11 @@ export function decorateShards(shards, nodes) {
         });
       }
     }
-    return capitalize(shard.state.toLowerCase());
+    let retMsg = 'Shard ' + shard.shard + ' ' + shard.state.toLowerCase();
```

Comment:
We probably need to localize this unfortunately. [This guide](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md#vanilla-js) might help but ping me if you have questions

Reply:
Done! Thanks for the suggestion.